### ISSUE TITLE
Launches Outlook every time I open OneNote

### ISSUE THREAD

**Issue by cspotcode** [stevencohn/OneMore#1517]
# Problem to Solve

_tl;dr opening OneNote launches Outlook, also freezing/hanging/performance issues, after upgrading to latest OneMore release._

For context, my Outlook is broken; I get a trio of error messages, one of which is the dreaded and inaccurate "system resources are critically low" which has no solutions anywhere online. I've tried over the years to fix this, never succeeded, but I don't use Outlook so it's not a big deal.

See "Screenshots" for the error messages, though they are beside the point, as far as I can tell.

This would not be a problem -- I don't use Outlook -- except OneMore seems to trigger it to open every time I launch OneNote.

I'm also seeing performance issues in OneNote which I'm pretty sure started after upgrading to the latest version.  I'm sorry I don't remember which version I was running before this.

I wonder, is Outlook launching automatically because OneMore is querying an Outlook API or linking against an Outlook DLL?

# To Reproduce

For me, it happens every time I open OneNote.  I'm not sure if there are other factors.

# Expected Behavior

Do not launch Outlook.

# Screenshots

![image](https://github.com/user-attachments/assets/ef7d0a72-8841-4aa0-b3a9-ac758f2726a4)
![image](https://github.com/user-attachments/assets/51c3a5f7-a19b-4943-a3aa-965ee08c1652)
![image](https://github.com/user-attachments/assets/cd5778e8-1253-4d23-ad35-23816788b74d)

# Environment (if applicable)

 - OneNote Version: 16.0.17726.20160 on Windows 11
 - OneMore Version: 6.5.0

**Comment by stevencohn** [comment]
To be clear, you think this is due to OneMore? Will this launching stop if you uninstall OneMore?

**Comment by cspotcode** [comment]
I tested again: it happens without OneMore.  False alarm, sorry!

This started at the same time as my installing OneMore 6.5.0 and the perf issues with the hashtag scan, so I incorrectly reasoned they might be related.  But trying to flag an Outlook task in OneNote causes it, not OneMore.

I'm guessing OneNote tries to talk to Outlook behind the scenes, but since my Outlook is broken, it goes through the bootup sequence which always shows me errors.
